source/api/pubsub.md

```diff
@@ -260,3 +260,45 @@ messages. When you change rooms by calling `Session.set('currentRoom',
 'newRoom')`, Meteor will subscribe to the new room's chat messages,
 unsubscribe from the original room's chat messages, and continue to
 stay subscribed to your private messages.
+
+## Publication strategies
+
+> The following features are available from Meteor 2.4 or `ddp-server@2.5.0`
+
+Once you start scaling your application you might want to have more control on how the data from publications is being handled on the client.
+There are three publications strategies:
+
+#### SERVER_MERGE
+`SERVER_MERGE` is the default strategy. When using this strategy, the server maintains a copy of all data a connection is subscribed to.
+This allows us to only send deltas over multiple publications.
+
+#### NO_MERGE_NO_HISTORY
+The `NO_MERGE_NO_HISTORY` strategy results in the server sending all publication data directly to the client.
+It does not remember what it has previously sent to it will not trigger removed messages when a subscription is stopped.
```

Comment:
I believe there is a typo here, this sentence is not clear enough.

`...previously sent to it will not trigger...`

Reply:
fixing